### PR TITLE
feat(*): expressions routes support [KM-20]

### PR DESCRIPTION
**Pull request by sumimakito**
# Summary

This PR adds support for creating/editing the Expressions routes in `@kong-ui-public/entities-routes`, and also:

- Adds Expressions schemas for the WS and WSS protocols
- Adds test cases to cover creating and editing both traditional and Expressions routes
- Updates the route form page in the sandbox to support testing the combination of different route flavors

KM-20

### CODE REVIEW

File: packages/entities/entities-routes/src/components/RouteFormExpressionsEditorLoader.vue [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+onMounted(async () => {
+  try {
+    await (await import('@kong-ui-public/expressions')).asyncInit
+    expressionInitialized.value = true
+  } catch (e) {
+    expressionInitError.value = true
+    console.error(e)
+  }
+})
+</script>
```
Comment (Leopoldthecoder) [inline packages/entities/entities-routes/src/components/RouteFormExpressionsEditorLoader.vue:55]: Should we replace plain text in this file with i18n entries?

File: packages/entities/entities-routes/sandbox/pages/RouteFormPage.vue [hunk 2]
```diff
@@ -91,3 +117,29 @@ const onUpdate = (payload: Record<string, any>) => {
   router.push({ name: 'route-list' })
 }
 </script>
+
+<style lang="scss" scoped>
+.sandbox-route-flavor-control {
+  :deep(.k-collapse) {
+    &.is-collapsed {
+      .k-collapse-heading {
+        margin-bottom: 0 !important;
```
Comment (aanchalm01) [inline packages/entities/entities-routes/sandbox/pages/RouteFormPage.vue:126]: Can we please use tokens everywhere in the styles ?

Comment (sumimakito) [reply packages/entities/entities-routes/sandbox/pages/RouteFormPage.vue]: Thanks. I've updated the PR.

File: packages/entities/entities-routes/src/components/RouteFormExpressionsEditorLoader.vue (hunk 1 above)
Comment (sumimakito) [reply packages/entities/entities-routes/src/components/RouteFormExpressionsEditorLoader.vue:58]: Thanks. I've replaced these strings with i18n resources.

**Review by Leopoldthecoder — DISMISSED** [review]
LGTM

**Comment by sumimakito** [comment]
Marking this PR as a draft to test these packages in KM and GM.

**Review by Leopoldthecoder — APPROVED** [review]
Manually tested and LGTM

**Review by Leopoldthecoder — APPROVED** [review]

**Pull request merged**